rtgui/guiutils.h

```diff
@@ -430,6 +436,22 @@ class MyImageMenuItem : public Gtk::MenuItem
     const Gtk::Label* getLabel () const;
 };
 
+class MyProgressBar : public Gtk::ProgressBar
```

Comment:
Yes, I was about to create a `widgets` subdir for our custom widgets. `MyCombobox`, `MyProgressBar`, etc... are more utility class than new widgets, so should we move them to `widgets`, create a separate `utility` subdir ?